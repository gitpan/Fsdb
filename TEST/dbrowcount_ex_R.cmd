prog='dbrowcount'
in=TEST/dbcolstats_ex_R.in
cmp='diff -c '
