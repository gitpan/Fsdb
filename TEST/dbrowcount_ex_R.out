#fsdb n
6
#  | /home/johnh/BIN/DB/dbrow 
#  | /home/johnh/BIN/DB/dbcol event clock
#  | dbrowdiff clock
#  | /home/johnh/BIN/DB/dbcol absdiff
#   | dbfilealter -R C
#   | dbrowcount
